src/BizHawk.Common/checksums/CRC32.cs

```diff
@@ -66,69 +83,41 @@ private static uint gf2_matrix_times(ReadOnlySpan<uint> mat, uint vec)
 				matIdx++;
 			}
 			return sum;
-		}
-
-		private uint _current = 0xFFFFFFFFU;
-
-		/// <summary>The raw non-negated output</summary>
-		public uint Current
-		{
-			get => _current;
-			set => _current = value;
-		}
-
-		/// <summary>The negated output (the typical result of the CRC calculation)</summary>
-		public uint Result => ~_current;
-
-		public void Add(byte datum)
-		{
-			_current = CRC32Table[(_current ^ datum) & 0xFF] ^ (_current >> 8);
-		}
-
-		public void Add(ReadOnlySpan<byte> data)
-		{
-			foreach (var b in data)
-			{
-//				Add(b); // I assume this would be slower
-				_current = CRC32Table[(_current ^ b) & 0xFF] ^ (_current >> 8);
-			}
-		}
-
+		}
+
 		/// <summary>
 		/// Incorporates a pre-calculated CRC with the given length by combining crcs<br/>
 		/// It's a bit flaky, so be careful, but it works
 		/// </summary>
 		/// <remarks>algorithm from zlib's crc32_combine. read http://www.leapsecond.com/tools/crcomb.c for more</remarks>
 		public void Incorporate(uint crc, int len)
-		{
-			if (len == 0) return; // degenerate case
-
+		{
+			if (len == 0) return; // degenerate case
+
 			Span<uint> combinerState = stackalloc uint[64];
 			COMBINER_INIT_STATE.CopyTo(combinerState);
 			var even = combinerState.Slice(start: 0, length: 32);
-			var odd = combinerState.Slice(start: 32, length: 32);
-
-			// apply len zeros to crc1 (first square will put the operator for one zero byte, eight zero bits, in even)
+			var odd = combinerState.Slice(start: 32, length: 32);
+
+			// apply len zeros to crc1 (first square will put the operator for one zero byte, eight zero bits, in even)
 			do
-			{
-				// apply zeros operator for this bit of len
+			{
+				// apply zeros operator for this bit of len
 				gf2_matrix_square(even, odd);
 				if ((len & 1U) != 0U) _current = gf2_matrix_times(even, _current);
-				len >>= 1;
-
-				// if no more bits set, then done
-				if (len == 0U) break;
-
-				// another iteration of the loop with odd and even swapped
+				len >>= 1;
+
+				// if no more bits set, then done
+				if (len == 0U) break;
+
+				// another iteration of the loop with odd and even swapped
 				gf2_matrix_square(odd, even);
 				if ((len & 1U) != 0U) _current = gf2_matrix_times(odd, _current);
-				len >>= 1;
-
-				// if no more bits set, then done
-			} while (len != 0U);
-
-			// finally, combine and return
-			_current ^= crc;
+				len >>= 1;
+			} while (len != 0U); // if no more bits set, then done
+
+			// finally, combine and return
+			_current ^= crc;

```

Comment:
no whitespace-only changes please

Reply:
I think most of that was Visual Studio being a butt here with newlines, not sure how I fix this.